main.py

```diff
@@ -479,25 +490,27 @@ def main(args):
                       'duration': duration_to_string(selector.duration),
                       'site': selector.site,
                       'client_provider': selector.client_provider,
-                      'metric': selector.metric,
-                      'mlab_project': selector.mlab_project,
+                      'client_country': selector.client_country,
+                      'metric': selector.metric
```

Comment:
(optional): This is out of scope of your change, but do we actually need this `thread_metadata` dictionary? It's just a `Selector` with certain fields converted to strings. It would be nice to pass a `Selector` around instead and trim down the parameter list of a lot of these 8+ parameter functions.